tests/integration/integration_test.py

```diff
@@ -522,3 +522,7 @@ def testSetStorageLocation__existing_storage_location():
     new_retrieved_setting = syn.getProjectSetting(proj, 'upload')
     assert_equals(new_storage_setting, new_retrieved_setting)
 
+def testMoveProject():
+    proj1 = syn.store(Project(name=str(uuid.uuid4()) + "testMoveProject-child"))
+    proj2 = syn.store(Project(name=str(uuid.uuid4()) + "testMoveProject-newParent"))
```

Comment:
use `schedule_for_cleanup(proj)` to clean up created entities in integration tests